packages/makeup-menu-button/dist/mjs/index.js

```diff
@@ -8,7 +8,7 @@ const defaultOptions = {
   buttonValueType: "text",
```

Comment:
I think one thing we might need to do is make it clearer that some of these options are only valid for the single select (i.e. role=menuitemradio) variant. Maybe we could move those into a sub mapping? Or any other ideas?

```
const defaultOptions = {
  customElementMode: false,
  expandedClass: "menu-button--expanded",
  menuSelector: ".menu-button__menu",
  buttonTextSelector: `.btn__text`,
  singleSelect: {
    buttonValueType: "text", // ["text", "icon", "both"],
    menuItemIconSelector: ".icon",
    menuItemButtonAriaLabelSelector: null,
  }
};
```

Reply:
Feel free to just create an issue for this and merge and we can come back to it later.

Reply:
Created a new issue - https://github.com/makeup/makeup-js/issues/146